src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,113 @@
+Assignment #5: Launch Checklist Form
+====================================
+
+Using our knowledge of forms, the DOM, and HTTP, our favorite space shuttle program has asked us to create a quick launch checklist.
+We have four fields that need to be filled out with vital information: the pilot's name, the co-pilot's name, the fuel levels, and the weight of the cargo.
+
+Our pilot, Chris, and the co-pilot, Blake, have been hard at work securing the cargo and filling the shuttle tank. All we need to do is use validation to ensure that we have all of the info for the space shuttle program and make sure no one prematurely launches the shuttle.
+
+Requirements
+------------
+
+Create a Launch Checklist Form for astronauts to fill out in preparation for launch.
+The form should do the following:
+
+1. Validate the user responses with ``preventDefault()`` to ensure the following:
+
+   a. The user has entered something for every field.
```

Comment:
Passive to active suggestion: How about, "The user entered...." for 1a and 1b?

Reply:
Done! Thanks!